### PR TITLE
options argument missing in some wolk.js functions

### PR DESCRIPTION
**Pull request by brucexhan**
when comparing master vs dev, getBlock/getNode/updateNode/getName/getAccount/getLatestBlockNumber/transfer function is missing the `options` argument. This PR is to search the whole repo and add the missing `options` argument.

### CODE REVIEW

File: lib/browser.js [hunk 1]
```diff
@@ -382,26 +382,26 @@ class WOLK {
     }
 
     // Blockchain operations
-    getBlock(blockNumber, callback) {
+    getBlock(blockNumber, options, callback) {
```
Comment (rodneywitcher) [inline lib/browser.js]: should we be passing options in to the "get" call instead of "{}" here?

Comment (brucexhan) [reply lib/browser.js]: we can. but we don't use options/headers in these get call functions. so I just followed the wolk.js in dev branch
https://github.com/wolkdb/wolkjs/blob/dev/apps/wolk.js#L640-L669
I can also replace the `{}` with `options` in these functions.

Comment (rodneywitcher) [reply lib/browser.js]: OK ... Yeah, I think we should go ahead and pass it through as that's the intention.  This way, if, in the future we do need/want to pass something in the headers, we have everything already setup.  Once that's changed, then I'll go ahead and merge.

Comment (brucexhan) [reply lib/browser.js]: done

**Pull request merged**